### DISCUSSION TITLE
poeditor localization disabled?

### DISCUSSION THREAD

**Discussion by xmgz** [writefreely/writefreely#857] [General]
I think it is important to this project make people feel comfortable using their own language. Tech people can read/understand english, but many communities are oriented to local audience and can not understand the (limited) options that writefreely offers to writing/reading.

Is **localization** still in the roadmap and important to you?

## No project at all to contribute to

in poeditor.com

> 
> There are no languages to translate in this localization project.
> 
> For more information, consider getting in touch with those in charge of managing the project.

but no way to contact you there.

### Steps to reproduce (if necessary)

go to https://poeditor.com/join/project/TIZ6HFRFdE  as referenced in Contributing guide

### Expected behavior

* being able to contribute with translation to my language and those contributions being regulary merged in the project 
* edit Contributing if needed

### moved?

is making a PR here the new way to contribute?
are you managing translations using another service? where? **thank you.**

**Comment by thebaer** [comment]
Thanks for bringing this up. It looks like poeditor changed their business model and we ran up against their limit for translated strings, so the project got archived.

I've just restored that, but we're still at the limit of what we can do on their free plan. I'll need to see if it's worth upgrading or switching to another service before we can accept more translations.

I'm changing this issue to a discussion, so we can continue discussing this and since it isn't a specific application bug.

**Comment by xabirequejo** [comment]
Hi,
I tried to contribute to WriteFreely by helping with localisation but I couldn't.

What's the current state regarding this issue? Have you implemented any other platform?
Is there any other way to contribute? PRs maybe?

Also, if Poeditor is not used anymore, I suggest you remove the links as I gave my data away for nothing.
Thanks
